src/lib/stable/canister_methods/on_low_wasm_memory.ts

```diff
@@ -5,20 +5,19 @@ import { Context, decoratorArgumentsHandler, OriginalMethod } from '.';
  *
  * @remarks
  *
- * The `onLowWasmMemory` entry point will be called when the canister is running low on WASM memory.
+ * The `onLowWasmMemory` entry point will be called when the canister is running low on Wasm memory.
  * This is a system method that allows canisters to handle low memory conditions gracefully.
  *
  * Only one `onLowWasmMemory` method is allowed per canister.
  *
+ * // TODO: verify the below info when onLowWasmMemory is fully supported on the IC (see https://forum.dfinity.org/t/how-to-verify-wasm-memory-threshold-is-set-correctly/40670)
  * - **State**: read-write
  *
- * - **Replication**: None // TODO: verify this
+ * - **Replication**: yes
  *
- * - **Async**: No // TODO: verify this
+ * - **Async**: no
  *
- * - **Instruction limit**: [200_000_000](https://internetcomputer.org/docs/current/developer-docs/smart-contracts/maintain/resource-limits) // TODO: verify this
- *
- * See [more documentation](https://docs.rs/ic-cdk/0.17.1/ic_cdk/attr.on_low_wasm_memory.html).
+ * - **Instruction limit**: [-](https://internetcomputer.org/docs/current/developer-docs/smart-contracts/maintain/resource-limits)
```

Comment:
Put the limit as 40 billion for now, I asked about these and that should be pretty close to accurate (I just don't know if ICP has launched a 50 billion instruction limit even though the documentation currently says 40 billion)

Reply:
Done